care/facility/models/asset.py

```diff
@@ -42,6 +45,8 @@ class Status(enum.Enum):
     support_name = models.CharField(max_length=1024, blank=True, null=True)
     support_phone = models.CharField(max_length=14, validators=[phone_number_regex], default="")
     support_email = models.EmailField(blank=True, null=True)
+    # Camera Details
+    camera = JSONField(default=dict, validators=[JSONFieldSchemaValidator(CAMERA_DETAILS)])
```

Comment:
We already have a meta JSON field in this model right? we can reuse that instead of having type-specific attributes.

Reply:
@vigneshhari What about validation? Is there a possibility of having polymorphic validation to the meta field, based on asset type or so?

Reply:
the `meta` json can have an optional key `camera` with a given data structure which can be enforced, we can go one level deep by overriding the `save` function and adding validation to only save when the `camera` key is present in `meta` as well.